app/assets/scss/graph.scss

```diff
@@ -1,6 +1,6 @@
-@import "../../../govuk/settings/all";
-@import "../../../govuk/tools/all";
-@import "../../../govuk/helpers/all";
+@import "../../../package/govuk/settings/all";
+@import "../../../package/govuk/tools/all";
+@import "../../../package/govuk/helpers/all";
 
 @include govuk-exports("idsk/component/graph") {
```

Comment:
potrebujeme tu robit vobec export? plus idsk/componet/graph ak to nie je uz komponent?